config/routes.rb

```diff
@@ -36,5 +36,16 @@
 
   get "/register", to: "users#new"
   post "/register", to: "users#create"
-  get "/profile", to: "users#show"
+
+  namespace :merchant do
+    get "/", to: "dashboard#index"
+  end
+
+  namespace :admin do
+    get "/", to: "dashboard#index"
+  end
+
+  namespace :profile do
+    get "/", to: "dashboard#index"
```

Comment:
can we route /users in here?

Reply:
Any URIs in this block will be prepended with "/profile". So if you add "/users" in this block, it will expand to mean "/profile/users". Or did you mean the `namespace :admin` block (because we will need "admin/users" for some stories)?